insert-symbol/insert-symbol.provider.css

```diff
@@ -0,0 +1,42 @@
+.k-icon.k-i-insertsymbol {
+    font-family: WebComponentsIcons;
+    font-size: 1.4em;
+}
+
+.k-i-insertsymbol::before {
+    font-family: WebComponentsIcons;
+    content: "Ω";
```

Comment:
Can we use the css entity for Omega here  - \03a9 :)

Reply:
is that the WebComponentsIcons symbol pointer?